src/schema/ghga.yaml

```diff
@@ -11,6 +11,7 @@ prefixes:
   GHGA: https://w3id.org/GHGA/
   SIO: http://semanticscience.org/resource/SIO_
   biolink: https://w3id.org/biolink/vocab/
+  efo: http://www.ebi.ac.uk/efo/
```

Comment:
* The prefix here should be `EFO`
* The base IRI  should be `http://www.ebi.ac.uk/efo/EFO_`